services/queue/src/api.js

```diff
@@ -935,19 +935,22 @@ let sleep20Seconds = () => {
 /** Claim any task */
 builder.declare({
   method: 'post',
-  route: '/claim-work/:provisionerId/:workerType',
+  route: '/claim-work/:taskQueueId(*)',
   name: 'claimWork',
   stability: APIBuilder.stability.stable,
   category: 'Worker Interface',
   scopes: { AllOf: [
-    'queue:claim-work:<provisionerId>/<workerType>',
+    'queue:claim-work:<taskQueueId>',
     'queue:worker-id:<workerGroup>/<workerId>',
   ] },
   input: 'claim-work-request.yml',
   output: 'claim-work-response.yml',
   title: 'Claim Work',
+  params: {
+    taskQueueId: /^[A-Za-z0-9_-]{1,38}\/[A-Za-z0-9_-]{1,38}$/,
+  },
```

Comment:
This can be added to the list of `params` in the call to `new APIBuilder` at the top of this file.